sensor_control/vision/symbols/src/FrameProc.h

```diff
@@ -0,0 +1,51 @@
+//Grabs frame from ROS, erode/dilate, converts to HSV, produces Red Green Blue Black binary frames;
+#ifndef FROM_PROC_H
+#define FROM_PROC_H
+
+#include "opencv2/core/core.hpp"
+#include "opencv2/highgui/highgui.hpp"
+#include "opencv2/imgproc/imgproc.hpp"
+#include "opencv2/opencv.hpp"
+
+#include "DebugWindow.h"
+
+using namespace cv;
+
+class FrameProc
+{
+	private:
+    //const int height = 480;
+		//const int width = 640;
+		static const int blur_kernel_size;
```

Comment:
Why are you making these static constants? It seems like you should be able to adjust these if you want to use this same object to process images with different parameters.